.devcontainer/Dockerfile

```diff
@@ -12,12 +12,13 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && pip install pipenv
 
 # [Optional] Uncomment the next lines to use go get to install anything else you need
-# USER vscode
-# RUN go get -x <your-dependency-or-tool>
-# USER root
+USER vscode
+RUN go install github.com/vektra/mockery/v2@v2.30.1
+RUN go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.53.3
+USER root
 
 # [Optional] Uncomment this line to install global node packages.
 # RUN su vscode -c "source /usr/local/share/nvm/nvm.sh && npm install -g <your-package-here>" 2>&1
 
 # Install xh
-RUN curl -sfL https://raw.githubusercontent.com/ducaale/xh/master/install.sh | XH_BINDIR=/usr/local/bin sh
\ No newline at end of file
+RUN curl -sfL https://raw.githubusercontent.com/ducaale/xh/master/install.sh | XH_BINDIR=/usr/local/bin sh
```

Comment:
nitpick: please remove the newline

Reply:
sure! done.

Reply:
Still there!

Reply:
emmmmmm

Reply:
ahhh, I found this guy. This guys is Goland IDE which add the new line =). done.